progress-bubble.html

```diff
@@ -18,15 +19,15 @@
             opacity: 0;
             transition: opacity 150ms;
         }
-        :host.calculation-ready svg,
-        :host.calculation-ready div{
+        :host(.calculation-ready) svg,
+        :host(.calculation-ready) div{
```

Comment:
Use `:host(.calculation-ready)` instead of `:host.calculation-ready` to make the rule apply in native Shadow DOM.